content/docs/command-reference/stage/add.md

```diff
@@ -0,0 +1,432 @@
+# stage add
+
+Helper command to add or update <abbr>stages</abbr> in `dvc.yaml`. Requires a
+name and a command.
+
+## Synopsis
+
+```usage
+usage: dvc stage add [-h] [-q | -v] -n <name> [-d <path>] [-o <path>]
+                     [-O <path>] [-p [<path>:]<params_list>] [-m <path>]
+                     [-M <path>] [--plots <path>] [--plots-no-cache <path>]
+                     [-w <path>] [-f]
+                     [--outs-persist <path>] [--outs-persist-no-cache <path>]
+                     [--always-changed] [--external] [--desc <text>]
+                    command
+
+positional arguments:
+  command               Command to execute
+```
+
+## Description
+
+`dvc stage add` is a helper for creating or updating
+[pipeline](/doc/command-reference/dag) stages in a `dvc.yaml` file (located in
+the current working directory).
+
+_Stages_ represent individual data processes, including their input and
+resulting outputs. They can be combined to capture simple data workflows,
+organize data science projects, or build detailed machine learning pipelines.
+
+A stage name is required and can be provided using the `-n` (`--name`) option.
+The other available [options](#options) are mostly meant to describe different
+kinds of stage [dependencies and outputs](#dependencies-and-outputs). The
+remaining terminal input provided to `dvc stage add` after `-`/`--` flags will
+become the required [`command` argument](#the-command-argument).
+
+`dvc stage add`, unlike `dvc run`, does not execute stage commands by default.
+`dvc repro` command can be used instead to execute them.
+
```

Comment:
If we're gonna remove the `run` ref in this PR as well (I think Ivan suggested that), maybe just say:
```suggestion
`dvc repro` can be used to execute pipelines after their stages have been defined.
```
And since it's such a short sentence, probably put it into some existing paragraph before.

Reply:
> And since it's such a short sentence, probably put it into some existing paragraph before.

I'll do this part.